compatibility/bazel_tools/daml_script/daml_script.bzl

```diff
@@ -80,13 +94,13 @@ def daml_script_test(compiler_version, runner_version):
         data = [
             compiled_dar,
         ],
-        runner = "//bazel_tools/client_server:runner",
-        runner_args = ["6865"],
-        server = daml_runner,
-        server_args = ["sandbox"],
-        server_files = [
-            "$(rootpath {})".format(compiled_dar),
-        ],
-        server_files_prefix = "--dar=" if versions.is_at_least("2.0.0", runner_version) else "",
+        runner = "//bazel_tools/client_server/with-upload-dar:runner",
```

Comment:
I slightly wonder if we should instead wrap the client and not the runner so that the runner can stay agnostic to what the client and server are. But happy either way.

Reply:
I think I'll look into that after moving the trigger tests over.

Reply:
No rush, this can also be after 2.0 or whenever.